specs/controller.test.js

```diff
@@ -0,0 +1,9 @@
+const app = require('../server/index');
+const supertest = require('supertest');
+
+const request = supertest(app);
+
+// This test fails because 1 !== 2
+it('Testing to see if Jest workd', () => {
+  expect(1).toBe(1);
```

Comment:
Good set up but you might want to build out your local tests a little more so that employers can see you know how to test with Jest.